fluentd/lib/filter_parse_json_field/lib/filter_parse_json_field.rb

```diff
@@ -72,5 +78,23 @@ def do_merge_json_log(record)
       record
     end
 
+    def do_replace_json_log(record)
+      json_fields.each do |merge_json_log_key|
+        if record.has_key?(merge_json_log_key)
+          value = record[merge_json_log_key].strip
+          if value[0].eql?('{') && value[-1].eql?('}')
```

Comment:
We should prefer `starts_with?` and `ends_with?` https://ruby-doc.org/core-2.5.1/String.html#method-i-start_with-3F